presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+* Introduced BigQuery Connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve ORC writer performance for string columns.
+* Add support for improving cache affinity hit rate by preventing repartitioning when nodes go down for a limited time. This can be enabled by setting the `internal-communication.memoize-dead-nodes-enabled` configuration property
+* Add support to enforce NOT NULL constraint by checking values for NOT NULL columns.
+
+
+JDBC Connector Changes
+______________________
+* Add support to accept username and password.
+
+Big Query Connector Changes
```

Comment:
Given how Google mentions it - https://www.google.com/search?q=big+query - `BigQuery` is a single word with capital B and Q - lets maintain their brand name

Reply:
Fixed in latest revision